pkg/sdk/common_types.go

```diff
@@ -96,9 +103,43 @@ func (row *propertyRow) toStringProperty() *StringProperty {
 }
 
 func (row *propertyRow) toIntProperty() *IntProperty {
+	var value *int
```

Comment:
can we add unit tests both to `toIntProperty` and `toBoolProperty`?

Reply:
ok